### PR TITLE
Finish v0.5.0

### PR DESCRIPTION
**Pull request by Strilanc**
Note that this is not going into master, but branching off into a release branch.

### CODE REVIEW

File: docs/noise.md [hunk 1]
```diff
@@ -101,7 +103,7 @@ such that each Pauli gate occurs with probability `p/3`.
 To construct channels, useful helpers are provided `cirq.asymmetric_depolarize`
 and `cirq.depolarize`.
 
-Another common case is when only a Pauli X (bit flip) can occur, or
+Another common case is when only a Pauli X (bit flip) can occur, or()
```
Comment (vtomole) [inline docs/noise.md]:
```suggestion
Another common case is when only a Pauli X (bit flip) can occur, or
```

Comment (Strilanc) [reply docs/noise.md]: Will fix in 0.6

File: .travis.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ language: python
 branches:
   only:
     - master
+    - v0.3.0-dev
```
Comment (vtomole) [inline .travis.yml]: What is this referring to?

Comment (Strilanc) [reply .travis.yml]: Just another branch that is in the repo.

Comment (vtomole) [reply .travis.yml]: Sure. Why this branch specifically?

Comment (Strilanc) [reply .travis.yml]: It was the only other dev branch. I can remove it.

**Pull request merged**